specification/communication/data-plane/CallAutomation/preview/2024-09-01-preview/communicationservicescallautomation.json

```diff
@@ -3364,6 +3426,23 @@
         }
       }
     },
+    "SummarizeCallRequest": {
+      "type": "object",
+      "properties": {
+        "operationContext": {
+          "description": "The value to identify context of the operation.",
+          "type": "string"
+        },
```

Comment:
Not sure what value would be valid for this `operationContext` (can user always use `a`?)

But since this pattern is used in existing APIs, I would assume user know what to use.

Reply:
value could be any string, it is for customer to make marks